### PR TITLE
final project 

### PR DESCRIPTION
**Pull request by misheck12**
In this milestone, we develop a stock market with the following features:

- [ ] The project is a single-page application (SPA) built with React and Redux.

- [ ] The data retrieved from the API should be stored in the Redux store.

live link https://www.loom.com/share/127635d3ee654be9b55656c642f3340d

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,52 @@
...
+> "Stock Market Metrics" is a website wher you can find information abput a cpmpany financial statement in the previous years.
 ## Built With
 
 - Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
```
Comment (arslanbisharat) [inline README.md:16]: - [x] your project is missing Live version link as requested in the project guidelines.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

Hi @misheck12, 

You've done well with the project requirements, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (misheck12) [reply README.md:16]: hey @arslanbisharat thank you for spotting this. i have added the live link

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @misheck12 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
 - Linter result is green ✔️
 - Commit messages are meaningful✔️
 - The pull request have a descriptive title telling you what changes are inside ✔️
 - The pull request have a short summary describing introduced changes in more detail ✔️

 ## Required Changes ♻️

- [x] Video Presentation:
- Please make sure the video contain a highlight about a piece of the code the student finds interesting or is proud of making

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by misheck12** [comment]
Hello @OLIPLICHE the changes have been implemented as requested thanks.

**Review by rukundoeric — APPROVED** [review]
Hi @misheck12,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed issues raised from the previous review. 💯 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**